pandas-stubs/core/series.pyi

```diff
@@ -1574,28 +1585,28 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     @overload
     def rolling(
         self,
-        window: int | BaseOffset | BaseIndexer,
+        window: int | _str | BaseOffset | BaseIndexer,
```

Comment:
changes to `rolling` should be in a different PR

Reply:
Fixed by rebase.